web/components/feed/feed-comments.tsx

```diff
@@ -267,224 +265,111 @@ function CommentStatus(props: {
   )
 }
 
-//TODO: move commentinput and comment input text area into their own files
-export function CommentInput(props: {
+export function ContractCommentInput(props: {
```

Comment:
Made the CommentInput  component generic, and created specific wrapper components for Contracts and Posts: ContractCommentInput, PostCommentInput.